frontend/src/components/sign/components/form-sign-up.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import { useForm } from "react-hook-form";
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import styles from './styles.module.scss';
+
+import { RegisterPayloadKey, IRegisterPayload, UserType, UserSex } from 'healthcare-shared'
```

Comment:
```suggestion
import { RegisterPayloadKey, IRegisterPayload, UserType, UserSex } from 'common/<interfaces/enums>'
```
Import only local source